dll/win32/shell32/dialogs/folder_options.cpp

```diff
@@ -278,13 +311,15 @@ Options_RunDLLCommon(HWND hWnd, HINSTANCE hInst, int fOptions, DWORD nCmdShow)
     switch(fOptions)
     {
         case 0:
-            ShowFolderOptionsDialog(hWnd, hInst);
+            ShowFolderOptionsDialog(PAGE_GENERAL);
+            break;
+
+        case 1: // Show taskbar options dialog
+            PostMessage(GetShellWindow(), WM_USER+22, fOptions, 0);
```

Comment:
What is this magic `WM_USER+22` message?

Reply:
I don't know the name of that message. It was already there in folder_options.cpp when I started and I tested it on Windows and it works there as well (0 opens the folder options and 1 opens the taskbar options).

Reply:
Then it may be of interest to have a define for `WM_USER+22` somewhere (with a made-up name, like `WM_SHOW_TASKBAR_OPTIONS_DLG` that can be also used in the other CDesktopBrowser.cpp file)